<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskWidget</class>
 <widget class="QWidget" name="TaskWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>181</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>181</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="taskTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font-size: 20px;</string>
   </property>
   <property name="text">
    <string>ZADANIE</string>
   </property>
  </widget>
  <widget class="QLabel" name="taskDescription">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>221</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>221</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Lorem ipsum</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Edytuj</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Usuń</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
  </widget>
  <widget class="QPushButton" name="subtaskButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="toolTip">
    <string>Dodaj podzadanie</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
  </widget>
  <widget class="QWidget" name="subtasksContainer" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>241</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>241</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="subtasksLayout"/>
   </widget>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Dodano przez:&lt;br/&gt;&lt;/span&gt;Imię i nazwisko&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
